### PR TITLE
Remove overlap and update qe-app info

### PR DESCRIPTION
**Pull request by unkcpz**
This is blocked by https://github.com/aiidalab/aiidalab-qe/pull/232.

If I understand correctly @csadorf comment on https://github.com/aiidalab/aiidalab-registry/issues/88#issuecomment-1123738458 about multiple releases of an app, in this PR I remove the duplicate metadata between two previous release. Moreover, I add a concise one for newly released `v22.05.0`.

### CODE REVIEW

**Comment by unkcpz** [comment]
Hmm... It seems I can't remove medatata for releases `v20.11.0^..v20.12.0` even it is the subset of the `v20.11.0^..v22.05.0`.
So every release is independent of each other?

File: apps.yaml [hunk 1]
```diff
@@ -43,7 +43,10 @@ nanoribbons:
         title: Empa nanotech@surfaces Laboratory - Graphene nanoribbons
 quantum-espresso:
   releases:
-    - url: 'git+https://github.com/aiidalab/aiidalab-qe.git@master:v20.11.0^..'
+    - url: 'git+https://github.com/aiidalab/aiidalab-qe.git@master:v22.05.0^..'
+      metadata:
```
Comment (yakutovicha) [inline apps.yaml:47]: If you provide metadata under an URL - it should be complete, I believe. You can't provide just the `state` key, but all the keys. To my understanding, it should be either only URL or URL + **complete** metadata.

@csadorf correct me if I am wrong

**Comment by yakutovicha** [comment]
Another related suggestion: what if we just discontinue the old releases?

File: apps.yaml (hunk 1 above)
Comment (csadorf) [reply apps.yaml:47]: Yes, that's correct. If you provide metadata for release line, it needs to be complete, it is not merged.

**Comment by csadorf** [comment]
> Another related suggestion: what if we just discontinue the old releases?

Certainly a viable option. But is there really a **need** for this? Is it to reduce registry maintenance burden?

**Comment by yakutovicha** [comment]
> > Another related suggestion: what if we just discontinue the old releases?
> 
> Certainly a viable option. But is there really a **need** for this? Is it to reduce registry maintenance burden?

There is no actual need, but the versions that are <= 21.11 are quite limited in functionality. They also request a beta version of the AWB - which brakes the aiidalab environment.

Last but not least: the registry file can be quite reduced if we remove this:

https://github.com/aiidalab/aiidalab-registry/blob/78fbf6274a97bbd303b025d102ffe2e1a51b8f4e/apps.yaml#L57-L72

**Comment by unkcpz** [comment]
Hi @yakutovicha, do you have any idea why the build test failed? Is that still the `categories` format reason?

**Comment by unkcpz** [comment]
The page build is failed because of `categories` (again), which will be solved by https://github.com/aiidalab/aiidalab/pull/325

**Comment by unkcpz** [comment]
The tests should be fixed after https://github.com/aiidalab/aiidalab-registry/pull/100

**Comment by unkcpz** [comment]
@superstar54 @yakutovicha, the problem was fixed in `aiidalab` package, and the build test passed. Can you have a look at it.

**Comment by yakutovicha** [comment]
Sorry for ignoring this PR. Let's get it done 😄 👍

**Review by yakutovicha — APPROVED** [review]
I like very mich how simple it gets 👍

**Pull request merged**